packages/eslint-plugin/src/rules/padding-line-between-statements.ts

```diff
@@ -595,6 +595,7 @@ export default util.createRule<Options, MessageIds>({
     },
     fixable: 'whitespace',
     hasSuggestions: false,
+    // intentionally a non-array schema to mirror the base rule
```

Comment:
while you are at it, can we fix/improve this schema here?


```diff
-      items: {
+      items: [{
        type: 'object',
        properties: {
          blankLine: { $ref: '#/$defs/paddingType' },
          prev: { $ref: '#/$defs/statementType' },
          next: { $ref: '#/$defs/statementType' },
        },
        additionalProperties: false,
        required: ['blankLine', 'prev', 'next'],
-      },
+      }],
```

Reply:
no  because that schema is actually not a tuple - it's an array!
https://eslint.org/docs/latest/rules/padding-line-between-statements#rule-details

```json
{
    "padding-line-between-statements": [
        "error",
        { "blankLine": LINEBREAK_TYPE, "prev": STATEMENT_TYPE, "next": STATEMENT_TYPE },
        { "blankLine": LINEBREAK_TYPE, "prev": STATEMENT_TYPE, "next": STATEMENT_TYPE },
        { "blankLine": LINEBREAK_TYPE, "prev": STATEMENT_TYPE, "next": STATEMENT_TYPE },
        { "blankLine": LINEBREAK_TYPE, "prev": STATEMENT_TYPE, "next": STATEMENT_TYPE },
        ...
    ]
}
```